terracumber_config/tf_files/SUSEManager-Test-Orion-NUE.tf

```
@@ -201,13 +201,13 @@ module "cucumber_testsuite" {
       image = "sles15sp3o"
     }
     debian-minion = {
-      name = "min-ubuntu2004"
-      image = "ubuntu2004o"
+      name = "min-ubuntu2204"
+      image = "ubuntu2204o"
       provider_settings = {
         mac = "aa:b2:93:01:00:7b"
       }
       additional_repos = {
-        Salt_repo = "https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/testing:/Ubuntu2004/Ubuntu_20.04/"
+        Salt_repo = "https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/testing:/ubuntu2204/Ubuntu_20.04/"
```

```suggestion
        Salt_repo = "https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/testing:/ubuntu2204/Ubuntu_22.04/"
```
could that be the correct link instead?

---

actually, it seems like it isn't. but the correct one is different anyway
```suggestion
        Salt_repo = "https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/testing:/Ubuntu2204/Ubuntu_22.04/"
```

---

Right!